services/galley/src/Galley/Run.hs

```diff
@@ -79,17 +85,15 @@ run opts = lowerCodensity $ do
     void $ Codensity $ Async.withAsync $ collectAuthMetrics (env ^. monitor) (aws ^. awsEnv)
   void $ Codensity $ Async.withAsync $ runApp env deleteLoop
   void $ Codensity $ Async.withAsync $ runApp env refreshMetrics
+  void $ Codensity $ Async.withAsync $ runApp env undefined
```

Comment:
I think that this should be removed. `runApp` is going to call `unApp` on the undefined value and pass it into runReaderT. This will fail when it is waited on.

```
Loaded GHCi configuration from /tmp/cabal-repl.-94973/setcwd.ghci
λ> import Control.Monad.Reader
λ> import Control.Concurrent.Async
λ> wait =<< async (runReaderT undefined ())
*** Exception: Prelude.undefined
CallStack (from HasCallStack):
  undefined, called at <interactive>:3:28 in interactive:Ghci3
λ> 
```

Reply:
yes, certainly the `undefined` needs to be refined.  :)